next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
Could you please explain this file to me?

Reply:
This file ensures Next.js types are picked up by the TypeScript compiler.
Ref: https://nextjs.org/docs/basic-features/typescript